classes/module/Module.php

```diff
@@ -758,7 +758,7 @@ public function enable($force_all = false)
             }
         }
 
-        if ($this->getOverrides() != null) {
+        if ($force_all && $this->getOverrides() != null) {
```

Comment:
It looks like this will prevent overrides from being installed when you enable a module on a single shop.